transports/http/endpoints/api/access/access_endpoints_test.go

```diff
@@ -46,39 +46,16 @@ func TestAccessEndpointWithWrongAuthHeader(t *testing.T) {
 	}
 }
 
-// Tests the GET /access endpoint with global auth token.
```

Comment:
Why removed?

Reply:
Commented in a commit where this specific test was removed - I removed global auth token from config.

Reply:
Too long to explain :D had a different idea about auth token.

TL;DR:
Reverted this test.